docs/sources/panels-refactor/_index.md

```diff
@@ -0,0 +1,16 @@
++++
+title = "Panels refactored"
```

Comment:
Why is there a subdir called `panels-refactor`?

Reply:
updated this, resolved.